### PR TITLE
Allow setting logo in the admin config

### PR DESCRIPTION
**Pull request by hellcp-work**
Lets you change the logo in the frontend
Screenshot from 2024-04-17 12-48-36

For reviewers: https://obs-reviewlab.opensuse.org/hellcp-work-logo-upload-ui/configuration

### CODE REVIEW

**Review by krauselukas — CHANGES_REQUESTED** [review]
image
@hellcp-work I get an error when I try to update the logo

**Comment by eduardoj** [comment]
> @hellcp-work I get an error when I try to update the logo

@krauselukas, the error was related to the cache. I added a couple of commits. It should work now.

Just a couple of things:
- Will we provide a way to reset the image to the default?
- I tried to upload an SVG file and it didn't work.

File: src/api/app/models/configuration.rb [hunk 1]
```diff
@@ -97,9 +98,7 @@ def write_to_backend
   end
 
   def invalidate_cache
-    Rails.cache.fetch('configurations/1', expires_in: 1.day, force: true) do
-      self
-    end
+    Rails.cache.delete('configurations/1')
```
Comment (krauselukas) [inline src/api/app/models/configuration.rb:101]: @eduardoj with the switch from `fetch` to `delete` we will end up with an empty cache. Shouldn't we write it to the cache afterwards?

Comment (krauselukas) [reply src/api/app/models/configuration.rb:101]: because the fetch used `force` to cause rewrite, this is not given anymore

Comment (krauselukas) [reply src/api/app/models/configuration.rb:101]: alright

**Pull request merged**